backend/README.md

```diff
@@ -1 +1,20 @@
-Additional instructions needed
+# Backend
+
+I purposely made the backend not connect with the frontend.
+For now you can manually get the access token in the steps below.
+
+## Getting Started
+
+Ensure you have `node v18.12.1` and `npm v9.1.1` installed
+
+1. `npm i`
+1. `touch .env`
+- Ask Justin for the environment variables
```

Comment:
It would be helpful to specify which environment variables should be in `.env`, in case a collaborator wants to use their own account.

Reply:
Related to above, also tell the users to set the redirect URI (ie. to `http://localhost:8000/api/spotify/callback`)